TES2_14-jira-babel Moving Test to JDBC

pratikzode opened the pull request:
Migrating the extension installcheck tests into the JDBC framework.
`bb installcheck-extensions --extension=babelfishpg_tsql`

Task: TES2_14-jira-babel
Signed-off-by: Pratik Zode [pzode@amazon.com](mailto:pzode@amazon.com)

pratikzode committed:
TES2_14-jira-babel Moving Test to JDBC

Moved babel_select_distinct_top.sql to JDBC

Task: TES2_14-jira-babel
Signed-off-by: pratikzode <pratik.zode@gmail.com>

macwoody commented on test/JDBC/input/babel_select_distinct_top.sql:
```diff
@@ -0,0 +1,25 @@
+--Check if the extension is installed
+Select * from pg_extension WHERE extname = 'babelfish_tsql';
```
don't need this line, others are good

pratikzode committed:
TES2_14 Moving Test to JDBC

Moved babel_select_distinct_top.sql ,babel_typecode.sql, babel_emoji.sql, babel_set_command.sql, babel_transaction.sql to JDBC

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
Merge remote-tracking branch 'upstream/BABEL_3_X_DEV' into TES2_14-jira-babel

pratikzode committed:
TES2_14 Modify babel_select_distinct_top.out

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

macwoody reviewed (commented):
Please also delete the old ones from repo

forestkeeper reviewed (commented):
can't do the merge . The commit msg didn't contain the right msg and signed off

forestkeeper reviewed (commented):
Will do the merge after changing the commit msg

pratikzode committed:
TES2_14 Moving Test to JDBC

Moved  babel_219,babel_collection,babel_datatype,babel_ddl,babel_delete1,babel_function,babel_like,babel_typecode,babel_uniqueidentifier to JDBC
And removed it from babelfish_extensions/contrib/babelfishpg_tsql/sql/test directory

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
TES2_14 Deleting from expected_create.out

Have deleted
-Could not find tests for function sys.space
-Could not find tests for function sys.stuff
-Could not find tests for function sys.sysdatetime
From test/python/output/sql_validation_framework/expected_create.out

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
TES2_14 Deleting files from expected/test

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
TES2_14 Modify Test Babel_datatype

Task: TES2_14
Signed-off-by: pratikzode <pratik.zode@gmail.com>

pratikzode committed:
Merge remote-tracking branch 'upstream/BABEL_3_X_DEV' into TES2_14-jira-babel

Signed-off-by: pratikzode <pratik.zode@gmail.com>
